librz/bin/format/pe/pe.h

```diff
@@ -69,6 +69,21 @@
 
 #define PE_SCN_ALIGN_MASK 0x00F00000
 
+typedef struct rz_bin_pe_reloc_block_t {
+	ut32 page_rva;
+	ut32 block_size;
```

Comment:
please give a short description of what these fields are.
Example of formatting

```suggestion
	ut32 page_rva; ///< PE RVA see doc https://bla
	ut32 block_size; ///< PE reloc block size for xxxxx
```

Reply:
I added a link to the base relocations documentation section of the specification, I will add detailed comments to each struct field too though